lang/llvm-14/Portfile

```diff
@@ -538,6 +531,20 @@ if {${subport} eq "clang-${llvm_version}"} {
     if { ${cxx_stdlib} eq "libstdc++" } {
         default_variants-append +libstdcxx
     }
+
+    post-configure {
+	# -Wl,-syslibroot referencing the macOS SDK must not appear when linking
+	# runtime libraries for non-macOS platforms, which are cross-built and
+	# used for cross-platform support. The clang build will provide the
+	# proper -isysroot for the platform in these cases. Remove the macOS
+	# SDK.
+        foreach rtl_dir {asan lsan stats tsan/rtl ubsan ubsan_minimal} {
+            set rtl [lindex [split ${rtl_dir} /] 0]
+            foreach rtl_os {ios iossim} {
+                reinplace "s|-Wl,-syslibroot,${configure.sdkroot}||" "${workpath}/build/projects/compiler-rt/lib/${rtl_dir}/CMakeFiles/clang_rt.${rtl}_${rtl_os}_dynamic.dir/link.txt"
```

Comment:
Is this something that we should also do for clang-15 and newer ?

Reply:
The ports for clang-15 and newer don’t build runtime libraries for ios and iossim at the moment, because

```
# Disable iOS support
# To Be reviewed with later versions
# https://reviews.llvm.org/D133273
configure.args-append -DCOMPILER_RT_ENABLE_IOS=OFF
```

I’m not sure exactly why this was disabled—if it was for this reason, then yes, sure. But I don’t know whether there’s another reason standing in the way. That’s why I only did it for clang-14.

Reply:
Ah yes, good point. It would be good to review that at some point but its not required for this PR. Thanks for your contribution...